src/PHPStan/SarifErrorFormatter.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file was copied from https://github.com/jbelien/phpstan-sarif-formatter/blob/d8cf03abf5c8e209e55e10d6a80160f0d97cdb19/src/SarifErrorFormatter.php,
+ * originally released under the MIT license.
+ *
+ * The code has been changed to export rules (without descriptions)
+ */
+
+namespace MongoDB\Laravel\PHPStan;
+
+use Nette\Utils\Json;
+use PHPStan\Command\AnalysisResult;
+use PHPStan\Command\ErrorFormatter\ErrorFormatter;
+use PHPStan\Command\Output;
+use PHPStan\File\RelativePathHelper;
+use PHPStan\Internal\ComposerHelper;
+
+use function array_values;
+
+/** @internal */
+class SarifErrorFormatter implements ErrorFormatter
```

Comment:
I don't like that we have this class in the package. Maybe create a 3rd party package that we add as 3rd party dev dependency?

Reply:
Or maybe you'll do a PR to integrate it into PHPStan directly?

Reply:
I'll point you to the comment at the top of the file; I started out with the 3rd party package but wanted to add a rule. Given that the 3rd-party package has little to no activity and little to no installs, there's also no point in relying on an external tool.

Reply:
Ok, so let's move the class into the `tests` directory, so that it's not in the package artefact.

Reply:
It's not a test, so I've gone ahead and excluded it via the .gitattributes file

Reply:
Moved.